src/main/index.js

```diff
@@ -995,14 +995,27 @@ function runApp() {
           return null
 
         case DBActions.PLAYLISTS.DELETE_VIDEO_ID:
-          await baseHandlers.playlists.deleteVideoIdByPlaylistId(data._id, data.playlistItemId)
+          await baseHandlers.playlists.deleteVideoIdByPlaylistId({
+            _id: data._id,
+            videoId: data.videoId,
+            playlistItemId: data.playlistItemId,
+          })
           syncOtherWindows(
             IpcChannels.SYNC_PLAYLISTS,
             event,
             { event: SyncEvents.PLAYLISTS.DELETE_VIDEO, data }
           )
           return null
 
+        case DBActions.PLAYLISTS.SET_QUICK_SAVE_TARGET_PLAYLIST:
```

Comment:
Could you please add this constant to the constants file, as it seems to be missing at the moment?

Reply:
Oops forgot to remove these unused code (which is why it's working without error and I didn't notice)